### ISSUE TITLE
branch: a11ystring-plugin

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#777]
To support https://github.com/phetsims/rosetta/issues/193

**Comment by zepumph** [comment]
This should be merged to master as soon as https://github.com/phetsims/rosetta/issues/193 is complete. Marking on-hold and assigning to myself.

**Comment by zepumph** [comment]
- [ ] When this is merged to master, then friction should be too, see https://github.com/phetsims/friction/issues/178

**Comment by zepumph** [comment]
This has been merged to master and the branch deleted, closing
